drf_spectacular/settings.py

```diff
@@ -9,6 +9,16 @@
     'SCHEMA_PATH_PREFIX': r'',
     'DEFAULT_GENERATOR_CLASS': 'drf_spectacular.generators.SchemaGenerator',
 
+    'SWAGGER_UI_DIST': '//unpkg.com/swagger-ui-dist@3.35.1',
+    'FAVICON_HREF': '//unpkg.com/swagger-ui-dist@3.35.1/favicon-32x32.png',
```

Comment:
for clarity call it `SWAGGER_UI_FAVICON_HREF`. also please restore the original version and just append the 2 new settings below `SWAGGER_UI_SETTINGS` for clarity